src/CNCULT2.CPP

```diff
@@ -377,15 +377,20 @@ RC TOPRAT::tp_SetCheckTimeSteps()	// initialize timesteps
 	tp_subhrDur = 1.f / tp_nSubSteps;			// subhr duration
 
 	// substep ticks = even shorter steps for e.g. HPWH simulation
-	tp_nSubhrTicks = max(1, 60 / tp_nSubSteps);
+	if (!IsSet(tp_nSubhrTicks)) { // Not set
+		tp_nSubhrTicks = max(1, 60 / tp_nSubSteps); // Default
+	}
+	else if (tp_nSubhrTicks != max(1, 60 / tp_nSubSteps)) {
+		warn("nSubhrTicks is in experimental mode");
+	}
 	tp_tickDurHr = 1. / (tp_nSubhrTicks * tp_nSubSteps);
 	tp_tickDurMin = tp_tickDurHr * 60.;
-
+	/*
 	if (tp_tickDurMin != 1. && bUsesTicks)
 		rc |= oer("nSubSteps=%d is invalid because it yields a non-integral substep duration (%0.3f min)."
 			   "\n    An integral duration is required when DHWSYS(s) are present.",
 			tp_nSubSteps, tp_tickDurMin);
-
+	*/
```

Comment:
We should deprecate the use of C-style comments, preferring "`//`". Although, better than both for this kind of code chunk is to preprocess it out with `#if 0`. (Other places in the code base that use this method, also add a comment after the `#if 0` that gives the date and reason for deprecating the code.)